packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -127,7 +124,7 @@ export class FunctionInfoGatherer implements ParametersGatherer<FunctionInfo> {
     }
 
     const language = await vscode.window.showQuickPick(
-      [LANGUAGE_JAVA, LANGUAGE_JAVASCRIPT],
+      [LANGUAGE_JAVA, LANGUAGE_JAVASCRIPT, LANGUAGE_TYPESCRIPT],
```

Comment:
I missed to see this in Yousef's review. Don't think we want to support typescript yet. This can be removed.

Reply:
@jag-sfdc Can you confirm that we'll keep it as is?

Reply:
Yes, we will keep this as is.